apps/easypid/src/crypto/pin.ts

```diff
@@ -36,126 +23,31 @@ export const deriveKeypairFromPin = async (agentContext: AgentContext, pin: Arra
     salt ?? TypedArrayEncoder.toUtf8String(pinSecret)
   )
 
-  const askarKey = AskarKey.fromSecretBytes({
-    algorithm: KeyAlgorithm.EcSecp256r1,
-    secretKey: new Uint8Array(TypedArrayEncoder.fromHex(pinSeed)),
-  })
-
-  const wallet = agentContext.wallet
-  assertAskarWallet(wallet)
-
-  await wallet.withSession(async (session) => {
-    const key = await session.fetchKey({
-      name: TypedArrayEncoder.toBase58(askarKey.publicBytes),
-    })
-    if (key) return
-    await session.insertKey({
-      name: TypedArrayEncoder.toBase58(askarKey.publicBytes),
-      key: askarKey,
-    })
-  })
-
-  return new Key(askarKey.publicBytes, KeyType.P256)
-}
-
-export const createPinDerivedEphKeyPop = async (
-  agent: EasyPIDAppAgent,
-  { aud, cNonce, deviceKey, pinDerivedEph }: { pinDerivedEph: Key; deviceKey: Key; cNonce: string; aud: string }
-) => {
-  const deviceKeyClaim = getJwkFromKey(deviceKey).toJson()
-
-  const payload = new JwtPayload({
-    aud,
-    additionalClaims: {
-      nonce: cNonce,
-      device_key: { jwk: deviceKeyClaim },
+  const { privateJwk } = transformPrivateKeyToPrivateJwk({
+    type: {
+      kty: 'EC',
+      crv: 'P-256',
     },
+    privateKey: TypedArrayEncoder.fromHex(pinSeed),
   })
 
-  const protectedHeaderOptions: JwsProtectedHeaderOptions = {
-    alg: 'ES256',
-    typ: 'pin_derived_eph_key_pop',
```

Comment:
Is this code not being used anymore, if it is, how is the `pin_derived_eph_key_pop` being set now?